src/com/palmergames/bukkit/config/ConfigNodes.java

```diff
@@ -285,7 +285,28 @@ public enum ConfigNodes {
 			"false",
 			"",
 			"# Setting this to true will set a nation's tag automatically using the first four characters of the nation's name."),
-	
+    NATION_DEF_TAXES(
+            "nation.default_taxes", 
+            "",
+            "",
+            "# Default tax settings for new nations."),
+    NATION_DEF_TAXES_TAX(
+			"nation.default_taxes.tax",
+			"0.0",
+			"",
+			"# Default amount of tax of a new nation. This must be lower than the economy.daily_taxes.max_nation_tax_amount setting."),
+    NATION_DEF_TAXES_TAXPERCENTAGE(
+            "nation.default_taxes.taxpercentage",
+            "true",

```

Comment:
Having this set to true would be a shift from the current default behaviour. This should be made false.